lxc/utils.go

```diff
@@ -44,6 +47,11 @@ func (p *ProgressRenderer) Done(msg string) {
 }
 
 func (p *ProgressRenderer) Update(status string) {
+	timeout := p.wait.Sub(time.Now())
```

Comment:
Subtract. So you take the end time and subtract the current time which gives you the duration between the two. Positive if not yet passed, negative otherwise.